pkg/agent/controller/traceflow/packetin.go

```diff
@@ -341,7 +345,26 @@ func (c *Controller) parsePacketIn(pktIn *ofctrl.PacketIn) (*crdv1beta1.Traceflo
 			}
 			ob.Action = crdv1beta1.ActionForwardedOutOfOverlay
 		} else if outputPort == gwPort { // noEncap
```

Comment:
Do we support Egress with NoEncap mode? If yes, shall we merge this branch into above one?